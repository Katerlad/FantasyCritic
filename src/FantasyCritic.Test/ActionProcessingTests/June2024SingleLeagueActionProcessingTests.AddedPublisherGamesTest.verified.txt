[
  {
    PublisherID: 63954675-e0f2-46ab-b14d-41f05e3be20d,
    MasterGameID: 3f74675f-2bee-4ec7-8ec7-d44eb5654835,
    GameName: Skate Story,
    CounterPick: false,
    SlotNumber: 12,
    BidAmount: 4
  },
  {
    PublisherID: 6bffef4c-9529-4f75-9a93-c383ffcd236a,
    MasterGameID: 470fe7f9-e48b-4694-9afe-f89794a86172,
    GameName: 33 Immortals,
    CounterPick: true,
    SlotNumber: 1,
    BidAmount: 4
  },
  {
    PublisherID: 788b4411-d078-4b03-9bab-efe034797738,
    MasterGameID: 9f7382a5-acf3-4798-a10b-e8e678d8d4f2,
    GameName: The Legend of Zelda: Echoes of Wisdom,
    CounterPick: false,
    SlotNumber: 13,
    BidAmount: 36
  }
]