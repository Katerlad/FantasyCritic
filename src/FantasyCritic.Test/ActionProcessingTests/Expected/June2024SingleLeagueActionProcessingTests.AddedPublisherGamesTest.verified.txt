[
  {
    PublisherID: 63954675-e0f2-46ab-b14d-41f05e3be20d,
    MasterGameID: 3f74675f-2bee-4ec7-8ec7-d44eb5654835,
    GameName: Skate Story,
    CounterPick: false,
    SlotNumber: 12,
    BidAmount: 4
  },
  {
    PublisherID: 788b4411-d078-4b03-9bab-efe034797738,
    MasterGameID: 9f7382a5-acf3-4798-a10b-e8e678d8d4f2,
    GameName: The Legend of Zelda: Echoes of Wisdom,
    CounterPick: false,
    SlotNumber: 13,
    BidAmount: 36
  },
  {
    PublisherID: 788b4411-d078-4b03-9bab-efe034797738,
    MasterGameID: db8b344a-0a54-4cbf-a62d-1690118efa6e,
    GameName: Super Mario Party Jamboree,
    CounterPick: false,
    SlotNumber: 6,
    BidAmount: 4
  }
]